code/game/objects/structures/girders.dm

```diff
@@ -87,10 +87,14 @@
 				reset_girder()
 
 	else if(istype(W, /obj/item/gun/energy/plasmacutter))
-		to_chat(user, "<span class='notice'>Now slicing apart the girder...</span>")
-		if(do_after(user,30/W.toolspeed))
-			if(!src) return
-			to_chat(user, "<span class='notice'>You slice apart the girder!</span>")
+		var/obj/item/gun/energy/plasmacutter/PC = W
+		to_chat(user, SPAN_NOTICE("You start lining up \the [PC] to the joints of \the [src]..."))
+		if(do_after(user, 2 SECONDS))
```

Comment:
don't think i need to log it, but this also reduces the amount of time it takes to cut girders